### DISCUSSION TITLE
Day Planner. Tasks with Clock in not showing in multiday or one day view as a block

### DISCUSSION THREAD

**Discussion by GMALEKKOS** [ivan-lednev/obsidian-day-planner#639] [Q&A] [answered]
Although I turned on "Show time tracker" I cannot see in the multiday or daily view tasks that are assigned with Clock in and Clock out.

**Comment by ivan-lednev** [comment, marked as answer]
Hello, @GMALEKKOS, make sure you include the task directory or tag in the Dataview filter (this works as a white list):
![image](https://github.com/user-attachments/assets/184fd30f-0291-4234-a1e5-d2d446049167)

**Reply by GMALEKKOS** [reply]
Thank you! Now is ok!
